pkg/edgeview/src/tcp.go

```diff
@@ -75,6 +77,11 @@ const (
 	tcpMaxMappingNUM  int           = 5
 	tcpIdleTimeoutSec float64       = 1800.0
 	tcpCheckTimeout   time.Duration = 300 * time.Second
+	tcpKubeEndpoint   string        = "localhost:6443"
+	kubeYamlFile      string        = "/run/.kube/k3s/user.yaml"
+	kubeConfigFile    string        = "kube-config-yaml"
```

Comment:
Any reason for the file to be `-yaml` and not `.yaml`?

Reply:
this used to be more suffix after that in the encrypted version. i'll change to '.yaml' then.